server/database/queries/patchPatientDataByIdQuery/index.js

```diff
@@ -0,0 +1,20 @@
+const connection = require('../../config/connection');
+
+const patchPatientDataByIdQuery = (
```

Comment:
I think it's good to change the shape of input parameters to be an object 👍🏻 
```
patchPatientDataByIdQuery({  firstName,
  lastName,
  phone,
  email,
  birthday,
  diseases,
  patientId,})
```
it makes the input operation easier, so you don't need to know exactly the order of them, but just only the names of these parameters.